api/tests/test_views.py

```diff
@@ -67,3 +68,72 @@ def test_filter_date_when_is_passing_wrong_format(self, api_client_authenticated
                 'O valor "%(value)s" tem um formato de data inválido.'
                 "Deve ser no formato  YYY-MM-DD."
             )
+
+
+class TestCityCouncilAttendanceListView:
+    url = reverse("city-council-attendance-list")
+
+    def test_should_list_city_council_attendance(self, api_client_authenticated):
+        presenca = baker.make_recipe("datasets.CityCouncilAttendanceList")
+        response = api_client_authenticated.get(self.url)
+        assert response.status_code == HTTPStatus.OK
+
+        data = response.json()
+        assert data[0]["date"] == presenca.date.strftime("%Y-%m-%d")
+        assert data[0]["description"] == presenca.description
+        assert data[0]["council_member"] == presenca.council_member
+        assert data[0]["status"] == presenca.status
+        assert len(data) == 1
+
+    @pytest.mark.parametrize(
+        "data, quantity_expected",
+        [
+            ({"query": "Competente da Silva"}, 3),
+            ({"start_date": "2020-9-15"}, 1),
+            ({"end_date": "2020-9-11"}, 1),
+            ({"start_date": "2020-9-11", "end_date": "2020-9-15"}, 3),
+            ({}, 3),
+        ],
+        ids=[
+            "filter_by_query",
+            "filter_by_start_date",
+            "filter_by_end_date",
+            "filter_by_range_date",
+            "filter_by_non",
+        ],
+    )
+    def test_should_filter_city_council_attendance(
+        self, api_client_authenticated, data, quantity_expected
+    ):
+
+        baker.make_recipe(
```

Comment:
Aqui você pode usar uma maneira mais curta de criar os objetos com dados aleatórios, já que não vai checar os valores retornados pela API:

```suggestion
        baker.make_recipe("datasets.CityCouncilAttendanceList", _quantity=3)
```

Reply:
Sim! Sem problemas.